### PR TITLE
Run doc-validator on Helm charts in addition to Mimir docs

### PR DESCRIPTION
**Pull request by jdbaldry**
Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

### CODE REVIEW

**Review by osg-grafana — APPROVED** [review]

**Comment by jdbaldry** [comment]
Think we might need to update the doc-validator image to support partial URIs. This will mean that we need to fix these up in the Mimir docs. That way we are fixing forward rather than going backwards by replacing these partial URIs with full URLs.

**Comment by jdbaldry** [comment]
To be clear, the latest doc-validator image will correctly support these partial URIs, we don't need any code changes in the tool itself. Only a bump of version in this repository.

**Comment by pracucci** [comment]
> To be clear, the latest doc-validator image will correctly support these partial URIs, we don't need any code changes in the tool itself. Only a bump of version in this repository.

Ok, can we bump the version directly in this PR? Any downside?

**Comment by jdbaldry** [comment]
I don't think so. I'll bump the version this afternoon and fix the resulting errors as a second commit.

File: docs/sources/mimir/release-notes/v2.3.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ The Mimir Helm chart is the best way to install Mimir on Kubernetes. As part of
 Notable enhancements follow. For the full list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
 
 - We've upgraded the MinIO subchart dependency from a [deprecated chart](https://helm.min.io/) to the [supported one](https://charts.min.io/). This creates a **breaking change** in how the administrator password is set. However, as the built-in MinIO is not a recommended object store for production use cases, this change did not warrant a new major version of the Mimir Helm chart.
-- [Query sharding](https://grafana.com/docs/mimir/latest/operators-guide/architecture/query-sharding/) is now enabled by default which should give you better performance on high cardinality metrics queries.
```
Comment (krajorama) [inline docs/sources/mimir/release-notes/v2.3.md:43]: [nit] these two could be relrefs

File: docs/sources/mimir/release-notes/v2.3.md [hunk 2]
```diff
@@ -40,8 +40,8 @@ The Mimir Helm chart is the best way to install Mimir on Kubernetes. As part of
...
-  - To compensate for the increased number of queries generated by query sharding, the [query scheduler](https://grafana.com/docs/mimir/latest/operators-guide/architecture/components/query-scheduler/) component is now enabled by default.
```
Comment (krajorama) [inline docs/sources/mimir/release-notes/v2.3.md:44]: [nit] this is the second one

**Review by krajorama — APPROVED** [review]
approved with nit comments

File: docs/sources/mimir/release-notes/v2.3.md [hunk 3]
```diff
@@ -40,8 +40,8 @@ The Mimir Helm chart is the best way to install Mimir on Kubernetes. As part of
...
+- [Query sharding]({{< relref "../operators-guide/architecture/query-sharding/ >}}) is now enabled by default which should give you better performance on high cardinality metrics queries.
```
Comment (krajorama) [inline docs/sources/mimir/release-notes/v2.3.md:43]:
```suggestion
- [Query sharding]({{< relref "../operators-guide/architecture/query-sharding/" >}}) is now enabled by default which should give you better performance on high cardinality metrics queries.
```

**Pull request merged**